test/lib/cli/commands/init.js

```diff
@@ -16,31 +12,27 @@ test.serial("Writes ui5.yaml to fs", async (t) => {
 	name: sample-app
 	type: application`;
 
-	const pathStub = sinon.stub(path, "resolve").returns(ui5YamlPath);
-	const fsHelperStub = sinon.stub(fsHelper, "exists").resolves(false);
-	const initStub = sinon.stub(init, "init").resolves({});
-	const safeDumpYamlStub = sinon.stub(jsYaml, "safeDump").returns(ui5Yaml);
-	const fsWriteFileStub = sinon.stub(fs, "writeFile").callsArgWith(2);
-	const lazyRequireStub = sinon.stub(initCommand, "lazyRequireDependencies").returns({
-		fsHelper: fsHelper,
-		init: init,
-		promisify: promisify,
-		path: path,
-		fs: fs,
-		jsYaml: jsYaml
+	mockRequire("path", {resolve: () => ui5YamlPath});
```

Comment:
Hm, this used to be done with Sinon. I would have used mockRequire only where necessary. I.e. whenever a module does *not* export an object.

I think I get that you only wanted to use one or the other, but in line 27 you are using (very rightly so) the more powerful sinon.stub anyways.

Using two mocking/stubbing frameworks in the same code can easily lead to confusion and errors. So I would try to limit the usage of one of them as much as possible. As Sinon has a more powerful API for creating stubs, I would prefer to use Sinon whenever possible.

Reply:
Fixed in 673d294